test/acceptance/gravity.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+var assert = require('assert');
+
+var Analysis = require('../../lib/analysis');
+
+var testConfig = require('../test-config');
+var QueryRunner = require('../../lib/postgresql/query-runner');
+
+describe('gravity analysis', function() {
```

Comment:
We've changed the way to implement these acceptance-tests, now we use a [test-helper](https://github.com/CartoDB/camshaft/blob/master/test/helper.js) to get results from any analysis. You can see [here](https://github.com/CartoDB/camshaft/blob/master/test/acceptance/line-to-single-point.js) an example to know how to use it.

Basically you need to remove `performAnalysis` function and use `test-helper` instead.